pkgs/servers/monitoring/icinga2/default.nix

```diff
@@ -0,0 +1,94 @@
+{ stdenv, lib, fetchFromGitHub, cmake, flex, bison, systemd, makeWrapper
+, boost, openssl, monitoring-plugins, mariadb-connector-c, postgresql, zlib
+# Databases
+, withMysql ? true, withPostgresql ? false
+# Features
+, withChecker ? true, withCompat ? false, withLivestatus ? false
+, withNotification ? true, withPerfdata ? true }:
+
+stdenv.mkDerivation rec {
+  pname = "icinga2";
+  version = "2.12.3";
+
+  src = fetchFromGitHub {
+    owner = "icinga";
+    repo = "icinga2";
+    rev = "v${version}";
+    sha256 = "0pq6ixv7d9bqys8qjxqq0jki3zncj8jdfavkp7qw125iyfjq48xk";
+  };
+
+  patches = [
+    ./remove-mkdir.patch
```

Comment:
This patch seems to being doing more than removing an mkdir. It's also convention to add a comment explaining why a patch is needed.